.github/workflows/release-ticket-creation.yml

```
@@ -0,0 +1,35 @@
+name: Release ticket creation on manual workflow trigger
+on:
+  workflow_dispatch:
+    inputs:
+      branch_name:
+        description: 'Release target branch name'
+        required: true
+
+jobs:
+  trigger-release-ticket-creation:
+    name: Trigger release ticket creation
+    runs-on: ubuntu-latest
+    # The branch name should starte with 'release/v'
```

👍

---

Oh my there's a typo 
```suggestion
    # The branch name should starts with 'release/v'
```